connector/src/main/java/org.apache.flink/connector/nebula/sink/NebulaBatchExecutor.java

```diff
@@ -35,22 +39,73 @@ public NebulaBatchExecutor(ExecutionOptions executionOptions, boolean isVertex,
         this.isVertex = isVertex;
         this.vidType = vidType;
         this.schema = schema;
+        init0();
+    }
+
+    public NebulaBatchExecutor(ExecutionOptions executionOptions, boolean isVertex,
+                               VidTypeEnum vidType, Map<String, Integer> schema, Class<? extends NebulaOutputFormatConverter<T>> converterClazz)
```

Comment:
please use  java's polymorphism but not clazz. Can we use the `NebulaOutputFormatConverter converter` as parameter?